### PR TITLE
convert yaml to nbt

### PR DESCRIPTION
**Pull request by cyilin**

### CODE REVIEW

File: src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/TempStorageRepo_v2.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import cat.nyaa.nyaacore.database.DataTable;
+import cat.nyaa.nyaacore.database.PrimaryKey;
+import cat.nyaa.nyaacore.utils.ItemStackUtils;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.List;
+import java.util.UUID;
+
+@DataTable("temporary_storage_v2")
+public class TempStorageRepo_v2 {
```
Comment (Librazy) [inline src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/TempStorageRepo_v2.java]: We may use some more descriptive name like `TempNbtStorageRepo`

**Review by Librazy — APPROVED** [review]

File: src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/TempStorageRepo_v2.java (hunk 1 above)
Comment (phoenixlzx) [reply src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/TempStorageRepo_v2.java]: Or use something like `NbtTempStorageRepo`, other than append `v2` should be better.

File: src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/MarketItem_v2.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    public int getAmount() {
+        return amount.intValue();
+    }
+
+    public void setAmount(int amount) {
+        this.amount = (long) amount;
+    }
+
+    @DataColumn("unit_price")
```
Comment (RecursiveG) [inline src/main/java/cat/nyaa/HamsterEcoHelper/utils/database/tables/MarketItem_v2.java]: NC Database should be able to process Double correctly.
No need to use getter/setter.

**Review by RecursiveG — CHANGES_REQUESTED** [review]
As long as NC database can process the java types correctly, we don't need those getter/setter.
Just set the fields as public.

**Comment by RecursiveG** [comment]
- Before merging this commit, I want to first deal with the TempStorage seperately. see #35 
- In any case, having a lot of `_v2` classes is unacceptable.

**Pull request closed without merging**